src/lib/domains/agentic/templates/README.md

```
@@ -40,21 +101,145 @@ interface PromptData {
 ```
 
 **Dependencies** (see `dependencies.json`):
 - `mustache` - Template rendering engine
 - `~/lib/domains/mapping` - For `MapItemType` enum
 
-**Internal Files** (prefixed with `_`):
-- `_prompt-builder.ts` - Core implementation (template lookup, data transformation, rendering)
-- `_system-template.ts` - SYSTEM tile template string and constants
+## File Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)